spec/lib/synapse/retry_policy_spec.rb

```diff
@@ -0,0 +1,150 @@
+require 'spec_helper'
+require 'synapse/retry_policy'
+
+describe "test with retry" do
+  include Synapse::RetryPolicy
+
+  it "default retry succeed" do
+    expected_attemptes = 1
```

Comment:
nit: extra `e` in `expected_attempts` (this occurs in a few places below as well)